### DISCUSSION TITLE
Presets

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#357] [Q&A] [answered]
Is the Export Presets (Application/Advanced/Presets) function no longer available?

**Comment by devemux86** [comment, marked as answer]
It was the old way, now it is deprecated.
Also the import of old .pref will be removed in the future.

Now this function is available in the profiles screen.

**Reply by airald** [reply]
This feature just should be used for global prefs, or also to save which maps and pois are downloaded and be able to load this extended presets to other devices.

For me it's enough exporting presets. Use on a motorized vehicle or walking just taping 3 times is extremely useful.
